packages/date-time-utilities/src/dateValues/timeConstants.ts

```diff
@@ -9,4 +9,6 @@ export const TimeConstants = {
   DaysInOneWeek: 7,
   MonthInOneYear: 12,
   HoursInOneDay: 24,
+  SecondsInOneHour: 60,
```

Comment:
1 hour has 3600 seconds... Either the constant should be named `SecondsInOneMinute` or its value should be changed to 3600.

Reply:
Ah great catch, thank you! Definitely meant `SecondsInOneMinute` here.

Reply:
Resolved by renaming to `SecondsInOneMinute`.